gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/listener/Listener.java

```diff
@@ -97,4 +90,34 @@ protected Listener(ListenerBuilder<?, ?> b) {
         this.entrypoints = b.entrypoints;
         this.servers = b.servers;
     }
+
```

Comment:
Maybe add a little comment with the ref to the lombock issue to help understand why we removed the annotation ?